Dockerfile

```diff
@@ -13,6 +13,7 @@ RUN apt-get update && apt-get -y install git \
            unzip
 
 RUN mkdir /JavaPrograms
+RUN mkdir /JavaPrograms/scripts
 RUN mkdir /code
```

Comment:
for future, if you do just `mkdir -p /JavaPrograms/scripts` this will create both directories with one line. This is a good strategy to have for Dockerfiles, because each line is a separate layer. It's good practice to reduce your runs into as few layers as possible. Eg:

```
RUN mkdir -p /JavaPrograms/scripts && mkdir /code
```
would be more optimal, although not completely necessary :)